### PR TITLE
Enhance unsupported properties test

### PR DESCRIPTION
**Pull request by KaiVolland**
## Description

This MR enhances the test for unsupported properties.

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] I am unsure (no worries, we'll find out)

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the [BSD 2-Clause License](https://github.com/geostyler/geostyler/blob/master/)
- [x] I have followed the [guidelines for contributing](https://github.com/geostyler/geostyler/blob/master/CONTRIBUTING.md)
- [x] The proposed change fits to the content of the [code of conduct](https://github.com/geostyler/geostyler/blob/master/CODE_OF_CONDUCT.md)
- [x] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [ ] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

File: src/OlStyleParser.ts [hunk 1]
```diff
@@ -1238,7 +1239,7 @@ export class OlStyleParser implements StyleParser<OlStyleLike> {
 
     const baseProps: OlStyleIconOptions = {
       src: isSprite(symbolizer.image) ? symbolizer.image.source as string : symbolizer.image as string,
-      crossOrigin: 'anonymous',
+      crossOrigin: null,
```
Comment (jansule) [inline src/OlStyleParser.ts:1242]: Can you elaborate on why this is needed?

Comment (KaiVolland) [reply src/OlStyleParser.ts:1242]: I actually don't know why this was set to "anonymous". The default value seems to be"null" when creating a minimum image style:
https://github.com/geostyler/geostyler-openlayers-parser/pull/827/files#diff-0c5b55a3fe5d9526721957d60430942cb5df5c522dbe670f1c6bded92110bad7R10-R14

Comment (jansule) [reply src/OlStyleParser.ts:1242]: This is include to ensure that the canvas is not tainted when requesting images from a foreign origin. I think it is better to keep it in.

**Review by jansule — APPROVED** [review]
Very nice!

**Pull request merged**